NOTICE

```diff
@@ -0,0 +1,26 @@
+Copyright 2024 Fastembed
+
+This product includes software developed by Fastembed
+
+This distribution includes the following Jina AI models, each with its respective license:
+- jinaai/jina-embeddings-v2-base-en
+    - License: apache-2.0
+- jinaai/jina-embeddings-v2-small-en
+    - License: apache-2.0
+- jinaai/jina-embeddings-v2-base-de
+    - License: apache-2.0
+- jinaai/jina-embeddings-v2-base-code
+    - License: apache-2.0
+- jinaai/jina-reranker-v1-tiny-en
+    - License: apache-2.0
+- jinaai/jina-reranker-v1-turbo-en
+    - License: apache-2.0
+- jinaai/jina-reranker-v2-base-multilingual
+    - License: cc-by-nc-4.0
+- jinaai/jina-colbert-v2
+    - License: cc-by-nc-4.0
+
+These models are developed by Jina  (https://jina.ai/) Limited and are subject to Jina AI's licensing terms.
+
+Additional Notes:
+This project also includes third-party libraries with their respective licenses. Please refer to the documentation of each library for details regarding its usage and licensing terms.
```

Comment:
AFAIK, we can avoid mentioning models with the same license and keep info only about more restrictive licenses.

Also I think it's better to write Qdrant rather than Fastembed

```suggestion
Copyright 2024 Qdrant

This product includes software developed by Qdrant.

Third-Party Content:
This distribution includes the following models from Jina AI, each subject to its respective license:

- **jinaai/jina-reranker-v2-base-multilingual**: CC BY-NC 4.0 (non-commercial use only)
- **jinaai/jina-colbert-v2**: CC BY-NC 4.0 (non-commercial use only)

Please refer to Jina AI's licensing terms at https://jina.ai/ for detailed information.
```